internal/core/metadata/rest_deviceprofile.go

```diff
@@ -68,40 +73,40 @@ func restAddDeviceProfile(
 		_, err := nameOp.Execute()
 		// The operator will return an ItemNotFound error if the DeviceProfile can not be found.
 		if err == nil {
-			httpErrorHandler.Handle(w, err, errorconcept.DeviceProfile.DuplicateName)
+			errorHandler.Handle(w, err, errorconcept.DeviceProfile.DuplicateName)
 			return
 		}
 
 		op := device_profile.NewAddValueDescriptorExecutor(r.Context(), vdc, loggingClient, dp.DeviceResources...)
 		err = op.Execute()
 		if err != nil {
-			httpErrorHandler.HandleOneVariant(w, err, errorconcept.NewServiceClientHttpError(err), errorconcept.Default.InternalServerError)
+			errorHandler.HandleOneVariant(w, err, errorconcept.NewServiceClientHttpError(err), errorconcept.Default.InternalServerError)
 			return
 		}
 	}
 
-	addDeviceProfile(dp, dbClient, w)
+	addDeviceProfile(dp, dbClient, w, errorHandler)
 }
 
 func restUpdateDeviceProfile(
 	w http.ResponseWriter,
 	r *http.Request,
 	loggingClient logger.LoggingClient,
-	dbClient interfaces.DBClient) {
+	dbClient interfaces.DBClient, errorHandler errorconcept.ErrorHandler) {
```

Comment:
Commented elsewhere in a separate context.

To follow folding pattern, this line with two parameters should be two lines with one parameter each.